CMakeLists.txt

```diff
@@ -46,6 +46,7 @@ endif()
 option(TRACY_ENABLE "Use tracy profiling" OFF) 
 option(TRACY_MEMORY_USAGE "Track memory allocation with tracy (expensive)" OFF)
 option(BUILD_TEST_CGAL "Build CGAL performance comparisons" OFF)
+option(TEST_HULL_PERFORMANCE "Test Hulling Algorithm" OFF)
```

Comment:
I think I'd rather build this under the `TEST_CGAL` flag instead of adding another.

Reply:
Alright, I will change that.